src/main/lrsql/admin/interceptors/ui.clj

```diff
@@ -14,8 +15,15 @@
    {:name ::get-env
     :enter
     (fn get-env [ctx]
-      (let [{url-prefix ::i/path-prefix
-             enable-stmt-html ::i/statement-html?} ctx]
-        (assoc ctx :response
-               {:status 200 :body {:url-prefix url-prefix
-                                   :enable-stmt-html (some? enable-stmt-html)}})))}))
+      (let [{url-prefix          ::i/path-prefix
+             enable-stmt-html    ::i/statement-html?
+             ?oidc-client-config ::oidc-i/client-config} ctx]
+        (assoc ctx
+               :response
+               {:status 200
+                :body
+                (cond-> {:url-prefix       url-prefix
+                         :enable-stmt-html (some? enable-stmt-html)}
+                  ?oidc-client-config
+                  (assoc :oidc
+                         ?oidc-client-config))})))}))
```

Comment:
Kind of weird that `enable-stmt-html` isn't added on a `cond->` branch while `?oidc-client-config` is, despite being nilable values?

Reply:
Enable-statement-html is a boolean, expected to be there, true or false. OIDC config is an optional feature with no meaning to clients launched without the intent to use OIDC